src/features/projects/components/ProjectSnippet.tsx

```diff
@@ -32,7 +32,8 @@ export default function ProjectSnippet({
     ? getImageUrl('project', 'medium', project.image)
     : '';
 
-  const { selectedPl, hoveredPl } = React.useContext(ProjectPropsContext);
+  const { selectedPl, hoveredPl, selectedSite } =
```

Comment:
We don't seem to be using `selectedSite` here.

Reply:
Addressed here - https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1601